gene/gene.go

```diff
@@ -31,16 +31,27 @@ const (
 // Gene is a processed version of a gtf record that enables easy
 // traversal and manipulation of genes on the genomic and mRNA levels.
 type Gene struct {
-	id           string       // Identifier for the transcript the Gene is derived from. In GTF this is the GeneID field.
-	startPos     int          // Genomic start position of the gene. This should use the coordinate system of the reference fasta, rather than Gene internal genomic coordinates
-	posStrand    bool         // True if gene is on the positive strand, False if on negative strand.
-	cdsStarts    []int        // The start position of each CDS. This value is stored in Gene genomic coordinates (slice index of genomeSeq)
-	cdsEnds      []int        // The end position of each CDS. This value is stored in Gene genomic coordinates (slice index of genomeSeq)
-	genomeSeq    []dna.Base   // The genomic sequence of the gene from 5' to 3'. NOTE: This field is reverse complemented relative to reference file if the gene is on the negative strand
-	cdnaSeq      []dna.Base   // The cDNA sequence of the gene from 5' to 3'.
-	featureArray []Feature    // FeatureArray is a slice with len(featureArray) = len(genomeSeq). The index of featureArray corresponds to the same index of genomeSeq. featureArray denotes the features listed above as negative values, or the cDNA pos using all values > 0.
-	orig         goGeneBackup // Copy of initial Gene state to enable the Reset() function.
-	changeLog    []diff       // Log of any mutations that have been performed on the Gene to enable the Reset() function.
+	id           string          // Identifier for the transcript the Gene is derived from. In GTF this is the GeneID field.
```

Comment:
Is it worth having a "symbol" field, where we could store the "human-readable" gene name (e.g. RUNX2)?

Reply:
added